up42/workflow.py

```diff
@@ -383,7 +385,9 @@ def _helper_run_job(
 
         logger.info("Selected input_parameters: %s.", input_parameters)
 
-        name = "_py"  # Enables recognition of python API usage.
+        if name is None:
+            name = self.info["name"]
+        name = f"{name}_py"  # Temporary recognition of python API usage.
```

Comment:
Is this still needed given we are already passing the custom headers in every call?

Reply:
Unfortunately yes, backend + bi solution is not yet completely figured out. We can just remove later without breaking anything.